docs/mounting.md

```diff
@@ -1,179 +1,202 @@
 
-The easiest way to set up credentials when running on [Google Compute
-Engine][gce] is to create your VM with a service account using the
-`storage-full` access scope. (See [here][gce-service-accounts] for details on
-VM service accounts.) When gcsfuse is run from such a VM, it automatically has
-access to buckets owned by the same project as the VM.
+        gcloud auth application-default login
+        gcloud auth login
+        gcloud auth list
 
-When testing, especially on a developer machine, credentials can also be
-configured using the [gcloud tool][]:
+   Alternatively, you can authenticate Cloud Storage FUSE by setting the --key-file flag to the path of a JSON key file, which you can download from the Google Cloud console. You can also set the GOOGLE_APPLICATION_CREDENTIALS environment variable to the path of the JSON key.
 
-    gcloud auth application-default login
-    gcloud auth login
+        GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json gcsfuse [...]
+When mounting with an fstab entry, use the key_file option:
 
-Alternatively, you can set the `GOOGLE_APPLICATION_CREDENTIALS` environment
-variable to the path to a JSON key file downloaded from the Google Developers
-Console:
+    my-bucket /mount/point gcsfuse rw,noauto,user,key_file=/path/to/key.json
 
-    GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json gcsfuse [...]
+When you create a Compute Engine VM, its service account can also be used to authenticate access to Cloud Storage FUSE. When Cloud Storage FUSE is run from such a VM, it automatically has access to buckets owned by the same project as the VM.
 
-When mounting with an fstab entry, you can use the `key_file` option. For example:
+# Basic syntax for mounting
 
-    my-bucket /mount/point gcsfuse rw,noauto,user,key_file=/path/to/key.json
+The base syntax for using Cloud Storage FUSE is:
 
-[gce]: https://cloud.google.com/compute/
-[gce-service-accounts]: https://cloud.google.com/compute/docs/authentication
-[gcloud tool]: https://cloud.google.com/sdk/gcloud/
-[app-default-credentials]: https://developers.google.com/identity/protocols/application-default-credentials#howtheywork
+    gcsfuse [global options] [bucket] mountpoint
 
-# Basic usage
+Where [global options] are optional specific flags you can pass (use gcsfuse --help), [bucket] is the optional name of your bucket, and ‘mountpoint’ is the directory on your machine that you are mounting the bucket to. For example:
+    
+    gcsfuse my-bucket /usr/me/myself
 
-## Mounting
+# Static Mounting
 
-There are two ways to mount gcsfuse. If you only use one known bucket, you may
-use static mounting. If you want to mount all the accessible buckets, you may
-use dynamic mounting.
+Static mounting means mounting a specific bucket. For example, say I want to mount the bucket “my-bucket” to the directory /usr/me/myself
```

Comment:
please replace /usr/me/myself to /path/to/mount/point at all places. (the user guide is also updated). Sorry to tell now but please also look in the docs published and correct there as well.

Reply:
fixed